tests/test_scheduler.py

```diff
@@ -102,6 +101,19 @@ def fixture_frontend():
     return FP if HAVE_FP else OFP
 
 
+def graphviz_present():
+    """
+    Test if graphviz is present and works
+    """
+    try:
+        import graphviz as gviz
+        callgraph = gviz.Graph()
+        callgraph.pipe()
+    except ImportError:
```

Comment:
Is `ImportError` really enough here? My understanding was that the import would succeed if the Python wrappers are installed but the actual graph generation was what failed. That throws a different exception, presumably?